src/models/CBCheckout.ts

```diff
@@ -46,12 +57,38 @@ export class CBCheckout {
       .join('&')
       .value();
   }
+
+  async checkforV1orV2() {
+    try {
+      const response = await fetch(
+        `https://${this.props.site}.chargebee.com/api/v2/configurations`,
+        {
+          method: 'GET',
+          headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+            'Authorization':
+              'Basic ' +
+              // base64.encode('test_rODRSf0YSfJJJzyy6bjUp16dZ9Kl0i1B:'),
+              base64.encode(`${this.props.apikey}`),
+          },
+        }
+      );
+      const json = await response.json();
+      console.log(json);
+      if (json.configurations[0].product_catalog_version === 'v2') return true;
+      else return false;
+    } catch (error) {
+      console.log(error);
```

Comment:
Should we handle this in case of error? Or perhaps a sensible default?

Reply:
return false is added.